README.md

```diff
@@ -23,19 +23,11 @@ You can also schedule them
 
 The [codemirror](http://codemirror.net) library is utilised as the editor and supports a range of languages and editor themes.
 
-## Downloads
-
-Download the *Scripting Helper* from the [Liferay Marketplace](https://www.liferay.com/marketplace/-/mp/application/25618082 "Liferay Scripting Helper").
-
-## Installation
-
-Copy the file `Scripting Helper.lpkg` package to the `deploy` folder of your installation.
-
 ## Building
 
 Step 1. Check out the source from GitHub:
 
-    % git clone https://github.com/campfire-digital-services/liferay-scripting-helper.git
```

Comment:
@acbenaissi  I'm not sure if there are any plans to support Liferay 7.4 in general by Campfire, though hypothetically let's say these do exist, then this specific part of your diff would need to be dropped for a merge into this repo. https://betterprogramming.pub/how-to-make-a-perfect-pull-request-3578fb4c112

Of course what you do in your own fork is up to you 😄